src/libs/actions/SignInRedirect.js

```diff
@@ -47,6 +47,10 @@ function redirectToSignIn(errorMessage) {
             if (activeClients && activeClients.length > 0) {
                 Onyx.set(ONYXKEYS.ACTIVE_CLIENTS, activeClients);
             }
+
+            // We must set the authToken to null so that signOut action is triggered across other clients
+            // https://github.com/Expensify/App/issues/4971#issuecomment-916101493
```

Comment:
NAB, I wonder if we need the link to the issue. If someone is curious about this we can use git blame and hopefully learn why some decision was made.